index.html

```diff
@@ -191,9 +192,57 @@ <h2>About</h2>
                 </div>
 
             </section>
+    
+<!-- contact section -->
+<section class="contact_section">
+    <div class="mobile_logo"></div>
+    
+      <h4 class="title-contact text_header_font">Get Started</h4>
 
-          
+      <p class="section-desc text_body_font">
+        Hello!I am a software developer! I can 
+        help you build a product, feature or website.
+        Take a look of my works. If you like what you see 
+        and have a project you need coded, don't hesitate 
+        and contact me.
+      </p>
+      <form action="put-link-here" method="post">
+          <input type="text" name="FullName" placeholder="Enter Your Name" maxlength="30" required>
+          <input type="email" name="Email" placeholder="Enter Your Email" required>
+          <textarea name="Message" placeholder="Enter Your Message"  maxlength="500" required></textarea>
+          <button class="" type="submit" name="submit-btn">Start collaboration</button>
+      </form>
 
+      <div class="bottom_row">
+        <a class="text-button text_body_font resume_color">
+            <p>Get my Resume</p>
+            <div class="arrow_upload"></div>
+        </a>
+
+        <div class="divicon">
+            <a href="" >
+            <div class="github_upload image_upload">
+                
+            </div>
+        </a>
+            <a href="">
+            <div class="twitter_upload image_upload">
+                
+            </div>
+        </a>
+            <a href="" >
+                <div class="linkin_upload image_upload">
+                    
+                </div>
+             </a>
+        
+
+        </div>
+
+      </div>
+      
+    
+  </section>

```

Comment:
- [ ] Kindly wrap all these in a footer tag instead of a section tag, if you check the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) the form is part of the footer